src/linux/cgroups2.cpp

```diff
@@ -964,6 +1005,17 @@ Result<Bytes> high(const string& cgroup)
   return internal::parse_bytelimit(*contents);
 }
 
+
+Try<Stats> stats(const string& cgroup)
+{
+  Try<string> contents = cgroups2::read<string>(cgroup, control::STAT);
+  if (contents.isError()) {
+    return Error("Failed to read 'memory.stat': " + contents.error());
+  }
+
+  return control::stat::parse(strings::trim(*contents));
```

Comment:
hm.. the parse function should know how to parse without the trimming, was this actually needed?